sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c

```diff
@@ -86,18 +86,18 @@ void _handle_snd_mc_ra(gnrc_netif_t *netif)
 
 void _snd_rtr_advs(gnrc_netif_t *netif, const ipv6_addr_t *dst, bool final)
 {
-#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_MULTIHOP_P6C)
-    _nib_abr_entry_t *abr = NULL;
-
-    DEBUG("nib: Send router advertisements for each border router:\n");
-    while ((abr = _nib_abr_iter(abr))) {
-        DEBUG("    - %s\n", ipv6_addr_to_str(addr_str, &abr->addr,
-                                             sizeof(addr_str)));
-        _snd_ra(netif, dst, final, abr);
+    if (IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_MULTIHOP_P6C) && gnrc_netif_is_6ln(netif)) {
```

Comment:
really `gnrc_netif_is_6ln()` or more specifically `gnrc_netif_is_6lr()`?

Reply:
I think we should only end up here if we are a router, but it's probably a good idea anyway.